### ISSUE TITLE
Reuse-block should maintain the `lang`-attribute of the parent content

### ISSUE THREAD

**Issue by tajakobsen** [ItemConsulting/lib-xp-item-blocks#7]
If the content being reused is in another language then the current content, a `[lang]` should be added to indicate to screen readers the real content.
